xml/ay_bigfile.xml

```diff
@@ -13869,6 +13869,74 @@ exit 0
          </para>
         </entry>
        </row>
+       <row>
+        <entry>
+         <para>
+          <literal>name</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Identifier of zone
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Used to identify zone. If it is not yet known zone, then new zone will be created. Mandatory parameter.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          <literal>short</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Short summary of zone
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Used mainly to document purpose of zone. Ignored for already existing zones. If not specified, name is used.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          <literal>description</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Description of zone
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Used mainly to document purpose of zone. Ignored for already existing zones. If not specified, name is used.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          <literal>target</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Default action
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Defines default action in zone if no rule matched. Possible values are "ACCEPT","%%REJECT%%","DROP" and "default". If not specified, "default" is used. For details about values please see https://firewalld.org/documentation/zone/options.html.
```

Comment:
For links, use the ```<link>``` element (and remove the 'please'): 
"For details about the values, see ```<link xlink:href="https://firewalld.org/documentation/zone/options.html"/>```."